src/avatar/__tests__/avatar.test.js

```diff
@@ -79,4 +79,12 @@ describe('Avatar', () => {
     triggerLoadError(wrapper);
     expect(wrapper.find(StyledInitials).text()).toBe('U');
   });
+
+  it('only renders initial from name if src is not provided', () => {
+    const name = 'user';
+    const wrapper = mount(<Avatar name={name} />);
+
+    triggerLoadError(wrapper);
```

Comment:
triggering an img src load error shouldn't be necessary. can it be removed and still pass ci? (may still need to call `wrapper.update()`)

```suggestion
```

Reply:
done.